README.md

```diff
@@ -0,0 +1,54 @@
+Basic React.js environment with ES6, Babel, and webpack-dev-server
+==========================
+
+
+run these command to get it working and visit localhost://8080
+```
+$ npm i
+$ npm run build
+$ webpack-dev-server
```

Comment:
You should point to a npm run command, rather than assuming that the users have webpack-dev-server installed globally.